### DISCUSSION TITLE
Parsing of magnetic moments and penalty energies from OSZICARs

### DISCUSSION THREAD

**Discussion by JonathanSchmidt1** [materialsproject/pymatgen#4066] [General]
Hi,
For some personal project I added some extra parsing capabilities for magnetic moments and penalty energies to the OSZICAR 
class I was wondering whether there is some general interest to add this to pymatgen. In that case I could add a test and open a pull request.

**Comment by janosh** [comment]
sounds like an excellent addition! PR would be much appreciated! 👍
